app/views/procedures/show.html.erb

```diff
@@ -1,8 +1,9 @@
 <% title @procedure.to_s  %>
+<% cache(@procedure) do %>
 <div class="panel panel-default" itemscope itemtype="https://schema.org/MedicalProcedure">
   <div class="panel-heading">
-    <%= @procedure.name %>
-    <%= link_to Pin.where(procedure_id: @procedure.id).count, pins_path(procedure: @procedure.id), { style: 'border-radius: 100%; background: #ffffcc; display:inline-block; padding:5px;' } %>
+    <h2><%= @procedure.name %></h2>
```

Comment:
imo the count is expensive and less readable / easy to understand than just a flat link, so for now changed it.